### ISSUE TITLE
Transform point cloud to world origin

### ISSUE THREAD

**Issue by hardikdava** [yzslab/nerfstudio#6]
**Is your feature request related to a problem? Please describe.**
- Current implementation renders the object below viewer planes and object is quite far then estimated.

**Describe the solution you'd like**
- Find center of the point cloud and translate to origin.

- Is it possible to do so? I would like to contribute but do not know where to implement this.

**Comment by yzslab** [comment]
Hi, sorry, I don't fully understand your problem. Would you mind providing some images or videos?

**Comment by hardikdava** [comment]
Thanks for your reply. As you can see here, the whole scene is below the axis plane. I want to move the scene to world origin. This can easily be achieved by computing mean of xyz and translate the whole scene to negative of mean xyz.

<img width="590" alt="Screenshot 2023-09-21 171654" src="https://github.com/yzslab/nerfstudio/assets/39372750/2fa696c6-8407-4614-8124-f39223d27726">

**Comment by hardikdava** [comment]
Okay. Thanks but I think I got it. I can do it when the point cloud is being loaded at the begining.

**Comment by yzslab** [comment]
> Thanks for your reply. As you can see here, the whole scene is below the axis plane. I want to move the scene to world origin. This can easily be achieved by computing mean of xyz and translate the whole scene to negative of mean xyz.
> 
> <img alt="Screenshot 2023-09-21 171654" width="590" src="https://user-images.githubusercontent.com/39372750/269666339-2fa696c6-8407-4614-8124-f39223d27726.png">

Hi, you can move the world origin by changing the `orientation_transform[:3, 3]` here: https://github.com/yzslab/nerfstudio/blob/a3f81112f21e11b3f29fd270091d2409db4d355b/nerfstudio/pipelines/gaussian_splatting_pipeline.py#L66

Here is the example: 
```python
        offset = torch.tensor(offset)  # offset = [x, y, z], set z to negative value if you want to move the plane down
        offset_transform = torch.eye(4)
        offset_transform[:3, 3] = offset

        # get orientation transform matrix
        if ref_orientation is not None:
            orientation_transform = self.get_orientation_transform_from_image(ref_orientation)
            orientation_transform = torch.matmul(orientation_transform, offset_transform)
            camera_pose_transform = torch.linalg.inv(orientation_transform)
        elif auto_reorient is True:
            camera_pose_transform = self.get_orientation_transform_by_up()
            camera_pose_transform[:3, 3] = -offset
            orientation_transform = torch.linalg.inv(camera_pose_transform)
        elif np.all(offset != 0.):
            orientation_transform = torch.eye(4)
            orientation_transform[:3, 3] = offset
            camera_pose_transform = torch.linalg.inv(orientation_transform)
```

You need to add some codes to load the point cloud and calculate the offset vector.
